test/test_prototype_transforms_functional.py

```diff
@@ -436,6 +439,29 @@ def test_dtype_and_device_convert_image_dtype(info, args_kwargs, device):
     assert output.device == input.device
 
 
+@pytest.mark.parametrize(
+    ("info", "args_kwargs"),
+    make_info_args_kwargs_params(
+        KERNEL_INFOS_MAP[F.posterize_image_tensor],
+        args_kwargs_fn=lambda info: info.reference_inputs_fn(),
+    ),
+)
+def test_posterize_image_tensor_float_vs_int(info, args_kwargs):
```

Comment:
This test makes sure that `posterize_image_tensor` is consistent for uint8 vs float32. We need this since the conversion to PIL and back introduces enough noise that float32 cannot be compared without crazy tolerances.

Maybe we should do this for all image kernels: test only uint8 against PIL and have a separate test that checks dtype consistency for these kernels. Maybe we can avoid some of the tolerance madness above.